site/app/views/calendar/CalendarView.php

```diff
@@ -40,11 +40,20 @@ public function showCalendar(CalendarInfo $info, array $courses, bool $in_course
         //Create list of courses and their term and get color
         $formatted_courses = [];
         $unformatted_courses = [];
+        $display_names = [];
         //$course_colors = [];
         foreach ($courses as $course) {
             $course_string = sprintf("%s %s", $course->getTitle(), $course->getTerm());
             array_push($formatted_courses, $course_string);
         }
+        $courseWithName = [];
+        foreach ($courses as $index => $course) {
+            $nameString = $course->getDisplayName() ?
+                        sprintf("%s %s", $course->getDisplayName(), $course->getTerm()) :
+                        sprintf("%s %s", $course->getTitle(), $course->getTerm());
```

Comment:
Why is `sprintf` being used here?  Can't you just do this with regular string concatenation?

Reply:
I used sprintf here because above code used sprintf. So i tried to keep the code in same format. But I changed them all to regular concatenation

Reply:
> Why is `sprintf` being used here? Can't you just do this with regular string concatenation?
I used sprintf here because above code used sprintf. So i tried to keep the code in same format. But I changed them all to regular concatenation